bin/node/test-runner-example/src/lib.rs

```diff
@@ -0,0 +1,259 @@
+        )
+    }
+
+    fn create_client_parts(
+        config: &Configuration,
+    ) -> Result<
+        (
+            Arc<TFullClient<Self::Block, Self::RuntimeApi, Self::Executor>>,
+            Arc<TFullBackend<Self::Block>>,
+            SyncCryptoStorePtr,
+            TaskManager,
+            InherentDataProviders,
+            Option<
+                Box<
+                    dyn ConsensusDataProvider<
+                        Self::Block,
+                        Transaction = sp_api::TransactionFor<
+                            TFullClient<Self::Block, Self::RuntimeApi, Self::Executor>,
+                            Self::Block,
+                        >,
+                    >,
+                >,
+            >,
+            Self::SelectChain,
+            Self::BlockImport,
+        ),
+        sc_service::Error,
+    > {
+        let (client, backend, keystore, task_manager) =
+            new_full_parts::<Self::Block, Self::RuntimeApi, Self::Executor>(config)?;
+        let client = Arc::new(client);
+
+        let inherent_providers = InherentDataProviders::new();
+        let select_chain = sc_consensus::LongestChain::new(backend.clone());
+
+        let (grandpa_block_import, ..) =
+            grandpa::block_import(client.clone(), &(client.clone() as Arc<_>), select_chain.clone())?;
+
+        let (block_import, babe_link) = sc_consensus_babe::block_import(
+            sc_consensus_babe::Config::get_or_compute(&*client)?,
+            grandpa_block_import,
+            client.clone(),
+        )?;
+
+        let consensus_data_provider = BabeConsensusDataProvider::new(
+            client.clone(),
+            keystore.sync_keystore(),
+            &inherent_providers,
+            babe_link.epoch_changes().clone(),
+            vec![(AuthorityId::from(Alice.public()), 1000)],
+        )
+            .expect("failed to create ConsensusDataProvider");
+
+        Ok((
+            client,
+            backend,
+            keystore.sync_keystore(),
+            task_manager,
+            inherent_providers,
+            Some(Box::new(consensus_data_provider)),
+            select_chain,
+            block_import,
+        ))
+    }
+
+    fn dispatch_with_root(call: <Self::Runtime as frame_system::Config>::Call, node: &mut Node<Self>) {
+        let alice = MultiSigner::from(Alice.public()).into_account();
+        let call = pallet_sudo::Call::sudo(Box::new(call)); // :D
+        node.submit_extrinsic(call, alice);
+        node.seal_blocks(1);
+    }
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use frame_system::{Call, Pallet};
+
+    #[test]
+    fn test_runner() {
+        let mut node = Node::<NodeTemplateChainInfo>::new().unwrap();
+        // seals blocks
+        node.seal_blocks(1);
+        // submit extrinsics
+        let alice = MultiSigner::from(Alice.public()).into_account();
+        node.submit_extrinsic(Call::remark((b"hello world").to_vec()), alice);
```

Comment:
```suggestion
        node.submit_extrinsic(frame_system::Call::remark((b"hello world").to_vec()), alice);
```

(it would be ambiguous otherwise)